core/cli.py

```diff
@@ -56,49 +56,13 @@ def seed_test():
     def drop():
```

Comment:
Do we want a test to cover this if we want to maintain it?

Reply:
I'm moving all CLI stuff to subsequent ticket, as some of the changes in there will have implications. But I may still just remove this stuff - it all depends how coupled it is to the new methods - I don't want them to be un-maintained as they are at the moment, like you say I will add tests if we keep